### DISCUSSION TITLE
Downloading Issue

### DISCUSSION THREAD

**Discussion by zzznelsonsm** [semiautomaticgit/SemiAutomaticClassificationPlugin#372] [General]
I am having difficulty downloading some Sentinel 2 images and get this error message. Can you give me some understanding on this?

Thank you!

SCP: failed download L1C_T16TEK_A047556_20240730_B11

**Comment by semiautomaticgit** [comment]
Hello @zzznelsonsm , usually these are temporary server issues.
You can try selecting the alternative product Sentinel-2_MPC.
